.storybook/preview.js

```diff
@@ -116,7 +116,7 @@ const withThemeProvider = (Story, context) => {
 export const decorators = [withThemeProvider, withPerformance]
 
 addons.setConfig({
-  // Some stories may set up keyboard event handlers, which can can be interfered
+  // Some stories may set up keyboard event handlers, which can be interfered
```

Comment:
This was caught by duplicate word detection, which is a fairly new feature in my tooling. Sometimes it's valuable, and sometimes it's annoying. I'm still gathering input based on experience...